### PR TITLE
Add Btc full block and BlockAlocator struct

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: include/veriblock/bfi/bitcoin/block.hpp [hunk 1]
```diff
@@ -34,6 +38,65 @@ struct BlockHeader : public BtcBlock {
   }
 };
 
+struct Block : public BlockHeader {
+  std::vector<Transaction> vtx{};
+  PopData popData{};
```
Comment (Warchant) [inline include/veriblock/bfi/bitcoin/block.hpp:43]: remove this. Block is a BTC block, not BTCSQ

File: include/veriblock/bfi/bitcoin/block.hpp [hunk 2]
```diff
@@ -4,14 +4,18 @@
 // file LICENSE or http://www.opensource.org/licenses/mit-license.php.
 
 #include "veriblock/bfi/bitcoin/serialize.hpp"
+#include "veriblock/bfi/bitcoin/transaction.hpp"
 #include "veriblock/pop/entities/btcblock.hpp"
+#include "veriblock/pop/entities/popdata.hpp"
```
Comment (Warchant) [inline include/veriblock/bfi/bitcoin/block.hpp:9]: use <> for #includes

File: include/veriblock/bfi/bitcoin/block.hpp (hunk 1 above)
Comment (Mr-Leshiy) [reply include/veriblock/bfi/bitcoin/block.hpp:43]: Ops, forget it :)

File: include/veriblock/bfi/bitcoin/block.hpp (hunk 2 above)
Comment (Mr-Leshiy) [reply include/veriblock/bfi/bitcoin/block.hpp]: fixed

**Review by Warchant — APPROVED** [review]

**Pull request merged**